moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -531,8 +531,8 @@ void PlanningSceneMonitor::newPlanningSceneCallback(const moveit_msgs::PlanningS
 
 void PlanningSceneMonitor::clearOctomap()
 {
-  if (scene_->getWorldNonConst()->removeObject(scene_->OCTOMAP_NS))
```

Comment:
For me this guard makes sense. Why did you remove it?

Reply:
I had issues where the octomap would still be displayed in Rviz, but not be considered for collisions. I assume that the MotionPlanning plugin's planning scene was not updated somehow.

Reply:
I couldn't reproduce those issues, so I reverted this change.